src/http-client.js

```diff
@@ -295,12 +302,14 @@ export default opts => {
     getInfo: () => info,
     ping: () => pubCall('/api/v3/ping').then(() => true),
     time: () => pubCall('/api/v3/time').then(r => r.serverTime),
-    exchangeInfo: () => pubCall('/api/v3/exchangeInfo'),
+    exchangeInfo: payload => exchangeInfo(pubCall, payload),
 
     book: payload => book(pubCall, payload),
     aggTrades: payload => aggTrades(pubCall, payload),
     candles: payload => candles(pubCall, payload),
 
+    getAll: payload =>
```

Comment:
call that method `accountCoins`, `getAll` doesn't mean anything

Reply:
I didn't choose that name, Binance did. In their API documentation it's called getAll and the endpoint is /sapi/v1/capital/config/getall

I agree it's not a good name, but I still prefer using Binance's naming over yet another name. Anyone trying to build an app will probably browser Binance's API docs and then search for the name they find there.